.github/workflows/test.yaml

```diff
@@ -1,11 +1,12 @@
 name: test
-on:
-  push:
```

Comment:
I can revert this, let me know. I can't get the MySQL runner to pass locally with [Act](https://github.com/nektos/act) but it doesn't play well with services so 🤷.